deepcell/applications/multiplex_segmentation.py

```diff
@@ -40,6 +40,100 @@
+                      }
+
+    return formatted_dict
+
+
+def deep_watershed_subcellular(model_output, compartment='whole_cell', min_distance=10,
+                               maxima_threshold=0.1, cell_threshold=0.1, exclude_border=False,
+                               small_objects_threshold=0):
+    """Postprocess model output to generate predictions for distinct cellular compartments
+
+    Args:
+        model_output (dict): Output from deep watershed model. A dict with a key corresponding to
+            each cellular compartment with a model prediction. Each key maps to a subsequent dict
+            with the following keys entries
+            - inner-distance: Prediction for the inner distance transform.
+            - outer-distance: Prediction for the outer distance transform
+            - fgbg: prediction for the foreground/background transform
+            - pixelwise: Prediction for the interior/border/background transform.
+        compartment: which cellular compartments to generate predictions for.
+            must be one of 'whole_cell', 'nuclear', 'both'
+        min_distance (int): Minimum allowable distance between two maxima to seed cells.
+        maxima_threshold (float): Threshold for maxima used to define watershed seeds.
+        cell_threshold (float): Threshold for mask used to define allowable area for watershed.
+        exclude_border (bool): Whether to include centroid detections at the border.
+        small_objects_threshold (int): Removes objects smaller than this size.
+
+    Returns:
+        numpy.array: Uniquely labeled mask for each compartment
+
+    Raises:
+        ValueError: for invalid compartment flag
+    """
+
+    valid_compartments = ['whole_cell', 'nuclear', 'both']
+
+    if compartment not in valid_compartments:
+        raise ValueError('Invalid compartment supplied: {}. '
+                         'Must be one of {}'.format(compartment, valid_compartments))
+
+    if compartment == 'whole_cell':
+        label_images = deep_watershed_mibi(outputs=model_output['whole_cell'],
+                                           min_distance=min_distance,
+                                           detection_threshold=maxima_threshold,
+                                           distance_threshold=cell_threshold,
+                                           exclude_border=exclude_border,
+                                           small_objects_threshold=small_objects_threshold)
+    elif compartment == 'nuclear':
+        label_images = deep_watershed_mibi(outputs=model_output['nuclear'],
+                                           min_distance=min_distance,
+                                           detection_threshold=maxima_threshold,
+                                           distance_threshold=cell_threshold,
+                                           exclude_border=exclude_border,
+                                           small_objects_threshold=small_objects_threshold)
```

Comment:
These two look almost the same.

```suggestion
    if compartment in {'whole_cell', 'nuclear'}:
        label_images = deep_watershed_mibi(outputs=model_output[compartment],
                                           min_distance=min_distance,
                                           detection_threshold=maxima_threshold,
                                           distance_threshold=cell_threshold,
                                           exclude_border=exclude_border,
                                           small_objects_threshold=small_objects_threshold)
```